src/Lykke.Common.ApiLibrary/Middleware/RequestUtils.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Http;
+
+namespace Lykke.Common.ApiLibrary.Middleware
+{
+    internal static class RequestUtils
+    {
+        public static async Task<string> GetRequestPartialBodyAsync(HttpContext context)
+        {
+            if (context.Request?.Body == null)
+            {
+                return null;
+            }
+
+            // request body might be already read at the moment 
+            if (context.Request.Body.CanSeek)
+            {
+                context.Request.Body.Seek(0, SeekOrigin.Begin);
+            }
+            
+            const int maxBytesToRead = 1024 * 64;
```

Comment:
Comment with explanation of this particular size?

Reply:
Added